tests/test_span_origin.py

```diff
@@ -0,0 +1,38 @@
+from sentry_sdk import start_transaction, start_span
```

Comment:
Let's put this file in `tests/tracing/test_span_origin.py`?

Reply:
moved!